payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/TextFieldUI.kt

```diff
@@ -147,6 +149,7 @@ internal fun TextField(
             }
             .semantics {
                 this.contentDescription = contentDescription
+                this.editableText = AnnotatedString("")
```

Comment:
It feels like content description should be describing what the field is, like the field requesting your name, and editableText should be the value entered by the user?

Reply:
Talkback already describes what the field is - that's a different property. ContentDescription is how we get talkback to read the contents of the textfield, by each number individually, as 4 2 4 2 instead of 4-thousand-2-hundred-twenty-two. Since we have contentDescription, the editableText is causing it to read the contents twice.